### PR TITLE
fix all the typos

### PR DESCRIPTION
**Pull request by dixudx**
/assign  @BenTheElder

Fix all the typos across the project. And with #6716, we could avoid new typos when merging a PR.

### CODE REVIEW

File: prow/plugins/approve/approve.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 			return nil, fmt.Errorf("invalid repo in enabledRepos: %q", repo)
 		}
 		opts := optionsForRepo(config, parts[0], parts[1])
-		approveConfig[repo] = fmt.Sprintf("Pull requests %srequire an associated issue.<br>Pull request authors %simplicitly approve their own PRs.", doNot(opts.IssueRequired), doNot(opts.ImplicitSelfApprove))
+		approveConfig[repo] = fmt.Sprintf("Pull requests %srequire an associated issue.<br>Pull request authors %simplicity approve their own PRs.", doNot(opts.IssueRequired), doNot(opts.ImplicitSelfApprove))
```
Comment (cjwagner) [inline prow/plugins/approve/approve.go]: This should be `implicitly` not `simplicity`.

**Comment by krzyzacy** [comment]
I love all the `TOOD` from @fejta :joy: 
/meow box

**Comment by krzyzacy** [comment]
/meow boxes
?

**Comment by fejta** [comment]
PHEW CRISIS AVERTED: <s>TOOD(fejta): fix cat plugin</s>

**Comment by BenTheElder** [comment]
AHHH the dog plugin needs to filter webm! :scream:

**Comment by fejta** [comment]
These cat images clearly aren't ready for production

**Comment by rmmh** [comment]
woof fix in #6730

File: prow/plugins/approve/approve.go (hunk 1 above)
Comment (BenTheElder) [reply prow/plugins/approve/approve.go]: Oh right: this should probably be `%s implicitly` or as-is..

**Comment by dixudx** [comment]
All the tests get passed.
/hold cancel

**Pull request merged**